db/migrate/20190711074433_change_course_accrediting_provider_id_to_code.rb

```diff
@@ -30,15 +30,5 @@ def change
         end
       end
     end
-
-    revert do
-      add_column :course, :accrediting_provider_id, :integer
-      add_foreign_key :course,
-                      :provider,
-                      column: :accrediting_provider_id,
-                      name: "FK_course_provider_accrediting_provider_id"
-      add_index :course, %w[accrediting_provider_id],
-                name: "IX_course_accrediting_provider_id"
-    end
```

Comment:
reverting the inverse of adding a revert. or something.
confusing but I think this is right to remove. qa will need resetting

Reply:
Ya, I buggered QA by trying to alter it in-place. Doh.